src/OrbitClientModel/CaptureDeserializerTest.cpp

```diff
@@ -221,12 +223,12 @@ TEST(CaptureDeserializer, LoadCaptureInfoOnCaptureStarted) {
 
         ASSERT_EQ(actual_instrumented_functions.size(), 1);
         ASSERT_TRUE(actual_instrumented_functions.contains(kInstrumentedFunctionId));
-        FunctionInfo actual_function_info =
+        const InstrumentedFunction& actual_function_info =
             actual_instrumented_functions.at(kInstrumentedFunctionId);
 
-        EXPECT_EQ(actual_function_info.name(), instrumented_function.name());
-        EXPECT_EQ(actual_function_info.pretty_name(), instrumented_function.pretty_name());
-        EXPECT_EQ(actual_function_info.size(), instrumented_function.size());
+        EXPECT_EQ(actual_function_info.function_name(), instrumented_function.pretty_name());
+        EXPECT_EQ(actual_function_info.file_path(), instrumented_function.loaded_module_path());
+        EXPECT_EQ(actual_function_info.file_offset(), instrumented_function.address() - load_bias);
```

Comment:
was the check for ".size()" removed on purpose?

Reply:
Yes, InstrumentedFunction message does not have size.